java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -779,6 +779,30 @@ public ColumnVector isNull() {
     }
   }
 
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
+   * TRUE for any entry that is Nan, and FALSE otherwise
```

Comment:
What happens if the input value is null?

Reply:
Will check and add tests.

Reply:
nulls are not nans is the behavior based on added test